### ISSUE TITLE
Kept alive Hero throws when the transition begins

### ISSUE THREAD

**Issue by LongCatIsLooong** [flutter/flutter#61929]
`localToGlobal` reports `(NaN, NaN)` when the render object is in the keep alive bucket

```dart
  testWidgets('kept alive Hero does not throw when the transition begins', (WidgetTester tester) async {
    final FocusNode focusNode = FocusNode();
    final GlobalKey<NavigatorState> navigatorKey = GlobalKey<NavigatorState>();

    await tester.pumpWidget(
      MaterialApp(
        navigatorKey: navigatorKey,
        home: Scaffold(
          body: ListView(
            children: <Widget>[
              Hero(
                tag: 'a',
                child: Material(
                  child: MediaQuery(
                    data: const MediaQueryData(),
                    child: TextField(focusNode: focusNode)
                  ),
                ),
              ),
              Container(height: 1000.0),
            ],
          ),
        ),
      ),
    );

    FocusScope.of(tester.element(find.byType(TextField))).requestFocus(focusNode);
    await tester.pump();

    await tester.drag(find.byType(ListView), const Offset(0.0, -1000.0));
    await tester.pump();

    navigatorKey.currentState.push(
      MaterialPageRoute<void>(
        builder: (BuildContext context) {
          return Scaffold(
            body: Center(
              child: Hero(
                tag: 'a',
                child: Material(
                  child: MediaQuery(
                    data: const MediaQueryData(),
                    child: TextField(focusNode: focusNode)
                  ),
                ),
              ),
            ),
          );
        },
      ),
    );
    await tester.pumpAndSettle();
  });
```

**Comment by TahaTesser** [comment]
<details> 
<summary>logs</summary> 

``` 
Running Gradle task 'assembleDebug'...                                  
Running Gradle task 'assembleDebug'... Done                        20.4s
✓ Built build/app/outputs/flutter-apk/app-debug.apk.
Installing build/app/outputs/flutter-apk/app.apk...                 8.2s
Waiting for SM M305F to report its views...                         11ms
Syncing files to device SM M305F...                              1,009ms

Flutter run key commands.
r Hot reload. 🔥🔥🔥
R Hot restart.
h Repeat this help message.
d Detach (terminate "flutter run" but leave application running).
c Clear the screen
q Quit (terminate the application on the device).
An Observatory debugger and profiler on SM M305F is available at: http://127.0.0.1:51142/Bv8ij7yETEo=/
D/SurfaceView(16625): surfaceChanged (1080,2214) 1 #8 io.flutter.embedding.android.FlutterSurfaceView{7059332 V.E...... ......ID 0,0-1080,2214}
I/flutter (16625): 00:00 +0: kept alive Hero does not throw when the transition begins
D/ViewRootImpl@1ede5e8[MainActivity](16625): stopped(false) old=true
D/SurfaceView(16625): windowStopped(false) true io.flutter.embedding.android.FlutterSurfaceView{7059332 V.E...... ........ 0,0-1080,2214} of ViewRootImpl@1ede5e8[MainActivity]
D/SurfaceView(16625): onWindowVisibilityChanged(0) true io.flutter.embedding.android.FlutterSurfaceView{7059332 V.E...... ........ 0,0-1080,2214} of ViewRootImpl@1ede5e8[MainActivity]
W/libEGL  (16625): EGLNativeWindowType 0x729521b810 disconnect failed
D/ViewRootImpl@1ede5e8[MainActivity](16625): Relayout returned: old=(0,0,1080,2340) new=(0,0,1080,2340) req=(1080,2340)0 dur=14 res=0x40001 s={true 489915998208} ch=false
D/ViewRootImpl@1ede5e8[MainActivity](16625): MSG_RESIZED_REPORT: frame=(0,0,1080,2340) ci=(0,83,0,126) vi=(0,83,0,126) or=1
D/OpenGLRenderer(16625): createReliableSurface : 0x729521b800, 0x721144e000
D/mali_winsys(16625): EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig, egl_winsys_surface **, EGLBoolean) returns 0x3000
D/ViewRootImpl@1ede5e8[MainActivity](16625): stopped(false) old=false
D/ViewRootImpl@1ede5e8[MainActivity](16625): Relayout returned: old=(0,0,1080,2340) new=(0,0,1080,2340) req=(1080,2340)0 dur=22 res=0x40001 s={true 489915998208} ch=false
D/ViewRootImpl@1ede5e8[MainActivity](16625): MSG_WINDOW_FOCUS_CHANGED 1 1
D/InputMethodManager(16625): prepareNavigationBarInfo() DecorView@e265fc[MainActivity]
D/InputMethodManager(16625): getNavigationBarColor() -855310
D/InputMethodManager(16625): prepareNavigationBarInfo() DecorView@e265fc[MainActivity]
D/InputMethodManager(16625): getNavigationBarColor() -855310
V/InputMethodManager(16625): Starting input: tba=com.example.master_flutter ic=null mNaviBarColor -855310 mIsGetNaviBarColorSuccess true , NavVisible : true , NavTrans : false
D/InputMethodManager(16625): startInputInner - Id : 0
I/InputMethodManager(16625): startInputInner - mService.startInputOrWindowGainedFocus
D/InputMethodManager(16625): prepareNavigationBarInfo() DecorView@e265fc[MainActivity]
D/InputMethodManager(16625): getNavigationBarColor() -855310
V/InputMethodManager(16625): Starting input: tba=com.example.master_flutter ic=null mNaviBarColor -855310 mIsGetNaviBarColorSuccess true , NavVisible : true , NavTrans : false
D/InputMethodManager(16625): startInputInner - Id : 0
I/flutter (16625): (The following exception is now available via WidgetTester.takeException:)
I/flutter (16625): ══╡ EXCEPTION CAUGHT BY RENDERING LIBRARY ╞═════════════════════════════════════════════════════════
I/flutter (16625): The following assertion was thrown during performLayout():
I/flutter (16625): BoxConstraints forces an infinite width and infinite height.
I/flutter (16625): These invalid constraints were provided to RenderIgnorePointer's layout() function by the following
I/flutter (16625): function, which probably computed the invalid constraints in question:
I/flutter (16625):   RenderStack.layoutPositionedChild (package:flutter/src/rendering/stack.dart:493:11)
I/flutter (16625): The offending constraints were:
I/flutter (16625):   BoxConstraints(biggest)
I/flutter (16625): 
I/flutter (16625): The relevant error-causing widget was:
I/flutter (16625):   MaterialApp file:///Users/tahatesser/AndroidStudioProjects/master_flutter/test/widget_test.dart:37:7
I/flutter (16625): 
I/flutter (16625): When the exception was thrown, this was the stack:
I/flutter (16625): #0      BoxConstraints.debugAssertIsValid.<anonymous closure>.throwError (package:flutter/src/rendering/box.dart:519:9)
I/flutter (16625): #1      BoxConstraints.debugAssertIsValid.<anonymous closure> (package:flutter/src/rendering/box.dart:559:21)
I/flutter (16625): #2      BoxConstraints.debugAssertIsValid (package:flutter/src/rendering/box.dart:567:6)
I/flutter (16625): #3      RenderObject.layout (package:flutter/src/rendering/object.dart:1679:24)
I/flutter (16625): #4      RenderStack.layoutPositionedChild (package:flutter/src/rendering/stack.dart:493:11)
I/flutter (16625): #5      _RenderTheatre.performLayout (package:flutter/src/widgets/overlay.dart:705:42)
I/flutter (16625): #6      RenderObject._layoutWithoutResize (package:flutter/src/rendering/object.dart:1639:7)
I/flutter (16625): #7      PipelineOwner.flushLayout (package:flutter/src/rendering/object.dart:896:18)
I/flutter (16625): #8      RendererBinding.drawFrame (package:flutter/src/rendering/binding.dart:404:19)
I/flutter (16625): #9      WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:881:13)
I/flutter (16625): #10     RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:286:5)
I/flutter (16625): #11     SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1114:15)
I/flutter (16625): #12     SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:1053:9)
I/flutter (16625): #13     LiveTestWidgetsFlutterBinding.handleDrawFrame (package:flutter_test/src/binding.dart:1430:13)
I/flutter (16625): #14     SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:969:5)
I/flutter (16625): #18     _invoke (dart:ui/hooks.dart:253:10)
I/flutter (16625): #19     _drawFrame (dart:ui/hooks.dart:211:3)
I/flutter (16625): (elided 3 frames from dart:async)
I/flutter (16625): 
I/flutter (16625): The following RenderObject was being processed when the exception was fired: _RenderTheatre#d3393 NEEDS-LAYOUT NEEDS-PAINT NEEDS-COMPOSITING-BITS-UPDATE:
I/flutter (16625):   needs compositing
I/flutter (16625):   creator: _Theatre ← Overlay-[LabeledGlobalKey<OverlayState>#b7023] ← _FocusMarker ← Semantics ←
I/flutter (16625):     FocusScope ← AbsorbPointer ← _PointerListener ← Listener ← HeroControllerScope ←
I/flutter (16625):     Navigator-[LabeledGlobalKey<NavigatorState>#46878] ← IconTheme ← IconTheme ← ⋯
I/flutter (16625):   parentData: <none> (can use size)
I/flutter (16625):   constraints: BoxConstraints(w=800.0, h=600.0)
I/flutter (16625):   size: Size(800.0, 600.0)
I/flutter (16625):   skipCount: 0
I/flutter (16625):   textDirection: ltr
I/flutter (16625): This RenderObject had the following descendants (showing up to depth 5):
I/flutter (16625):     onstage 1: RenderIgnorePointer#46de4
I/flutter (16625):       child: RenderBlockSemantics#f8c5e
I/flutter (16625):         child: RenderExcludeSemantics#7d690
I/flutter (16625):           child: RenderSemanticsGestureHandler#a13bd
I/flutter (16625):             child: RenderPointerListener#f07f2
I/flutter (16625):     onstage 2: RenderSemanticsAnnotations#b23de
I/flutter (16625):       child: RenderOffstage#ada19
I/flutter (16625):         child: RenderSemanticsAnnotations#7e2f5
I/flutter (16625):           child: RenderRepaintBoundary#188c3
I/flutter (16625):             child: RenderFractionalTranslation#0f1ae
I/flutter (16625):     onstage 3: RenderIgnorePointer#3900b
I/flutter (16625):       child: RenderBlockSemantics#d11a8
I/flutter (16625):         child: RenderExcludeSemantics#49351
I/flutter (16625):           child: RenderSemanticsGestureHandler#a8a81
I/flutter (16625):             child: RenderPointerListener#79a8c
I/flutter (16625):     onstage 4: RenderSemanticsAnnotations#697e1 NEEDS-PAINT
I/flutter (16625):       child: RenderOffstage#9c3fe NEEDS-PAINT
I/flutter (16625):         child: RenderSemanticsAnnotations#86a76 NEEDS-PAINT
I/flutter (16625):           child: RenderRepaintBoundary#b3483 NEEDS-PAINT
I/flutter (16625):             child: RenderFractionalTranslation#4aced NEEDS-PAINT NEEDS-COMPOSITING-BITS-UPDATE
I/flutter (16625):     onstage 5: RenderConstrainedBox#564ac
I/flutter (16625):     onstage 6: RenderLimitedBox#199fe
I/flutter (16625):       child: RenderConstrainedBox#b2002
I/flutter (16625):     onstage 7: RenderIgnorePointer#0c7ec NEEDS-LAYOUT NEEDS-PAINT NEEDS-COMPOSITING-BITS-UPDATE
I/flutter (16625):       child: RenderRepaintBoundary#0e41e NEEDS-LAYOUT NEEDS-PAINT NEEDS-COMPOSITING-BITS-UPDATE
I/flutter (16625):         child: RenderOpacity#15dd1 NEEDS-LAYOUT NEEDS-PAINT NEEDS-COMPOSITING-BITS-UPDATE
I/flutter (16625):           child: RenderPhysicalModel#6e59a NEEDS-LAYOUT NEEDS-PAINT NEEDS-COMPOSITING-BITS-UPDATE
I/flutter (16625):             child: _RenderInkFeatures#35c1b NEEDS-LAYOUT NEEDS-PAINT NEEDS-COMPOSITING-BITS-UPDATE
I/flutter (16625): ════════════════════════════════════════════════════════════════════════════════════════════════════
I/flutter (16625): (If WidgetTester.takeException is called, the above exception will be ignored. If it is not, then the above exception will be dumped when another exception is caught by the framework or when the test ends, whichever happens first, and then the test will fail due to having not caught or expected the exception.)
I/flutter (16625): ══╡ EXCEPTION CAUGHT BY RENDERING LIBRARY ╞═════════════════════════════════════════════════════════
I/flutter (16625): The following assertion was thrown during performLayout():
I/flutter (16625): BoxConstraints forces an infinite width and infinite height.
I/flutter (16625): These invalid constraints were provided to RenderIgnorePointer's layout() function by the following
I/flutter (16625): function, which probably computed the invalid constraints in question:
I/flutter (16625):   RenderStack.layoutPositionedChild (package:flutter/src/rendering/stack.dart:493:11)
I/flutter (16625): The offending constraints were:
I/flutter (16625):   BoxConstraints(biggest)
I/flutter (16625): 
I/flutter (16625): The relevant error-causing widget was:
I/flutter (16625):   MaterialApp file:///Users/tahatesser/AndroidStudioProjects/master_flutter/test/widget_test.dart:37:7
I/flutter (16625): 
I/flutter (16625): When the exception was thrown, this was the stack:
I/flutter (16625): #0      BoxConstraints.debugAssertIsValid.<anonymous closure>.throwError (package:flutter/src/rendering/box.dart:519:9)
I/flutter (16625): #1      BoxConstraints.debugAssertIsValid.<anonymous closure> (package:flutter/src/rendering/box.dart:559:21)
I/flutter (16625): #2      BoxConstraints.debugAssertIsValid (package:flutter/src/rendering/box.dart:567:6)
I/flutter (16625): #3      RenderObject.layout (package:flutter/src/rendering/object.dart:1679:24)
I/flutter (16625): #4      RenderStack.layoutPositionedChild (package:flutter/src/rendering/stack.dart:493:11)
I/flutter (16625): #5      _RenderTheatre.performLayout (package:flutter/src/widgets/overlay.dart:705:42)
I/flutter (16625): #6      RenderObject._layoutWithoutResize (package:flutter/src/rendering/object.dart:1639:7)
I/flutter (16625): #7      PipelineOwner.flushLayout (package:flutter/src/rendering/object.dart:896:18)
I/flutter (16625): #8      RendererBinding.drawFrame (package:flutter/src/rendering/binding.dart:404:19)
I/flutter (16625): #9      WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:881:13)
I/flutter (16625): #10     RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:286:5)
I/flutter (16625): #11     SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1114:15)
I/flutter (16625): #12     SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:1053:9)
I/flutter (16625): #13     LiveTestWidgetsFlutterBinding.handleDrawFrame (package:flutter_test/src/binding.dart:1430:13)
I/flutter (16625): #14     SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:969:5)
I/flutter (16625): #18     _invoke (dart:ui/hooks.dart:253:10)
I/flutter (16625): #19     _drawFrame (dart:ui/hooks.dart:211:3)
I/flutter (16625): (elided 3 frames from dart:async)
I/flutter (16625): 
I/flutter (16625): The following RenderObject was being processed when the exception was fired: _RenderTheatre#d3393:
I/flutter (16625):   needs compositing
I/flutter (16625):   creator: _Theatre ← Overlay-[LabeledGlobalKey<OverlayState>#b7023] ← _FocusMarker ← Semantics ←
I/flutter (16625):     FocusScope ← AbsorbPointer ← _PointerListener ← Listener ← HeroControllerScope ←
I/flutter (16625):     Navigator-[LabeledGlobalKey<NavigatorState>#46878] ← IconTheme ← IconTheme ← ⋯
I/flutter (16625):   parentData: <none> (can use size)
I/flutter (16625):   constraints: BoxConstraints(w=800.0, h=600.0)
I/flutter (16625):   size: Size(800.0, 600.0)
I/flutter (16625):   skipCount: 0
I/flutter (16625):   textDirection: ltr
I/flutter (16625): This RenderObject had the following descendants (showing up to depth 5):
I/flutter (16625):     onstage 1: RenderIgnorePointer#46de4
I/flutter (16625):       child: RenderBlockSemantics#f8c5e
I/flutter (16625):         child: RenderExcludeSemantics#7d690
I/flutter (16625):           child: RenderSemanticsGestureHandler#a13bd
I/flutter (16625):             child: RenderPointerListener#f07f2
I/flutter (16625):     onstage 2: RenderSemanticsAnnotations#b23de
I/flutter (16625):       child: RenderOffstage#ada19
I/flutter (16625):         child: RenderSemanticsAnnotations#7e2f5
I/flutter (16625):           child: RenderRepaintBoundary#188c3
I/flutter (16625):             child: RenderFractionalTranslation#0f1ae
I/flutter (16625):     onstage 3: RenderIgnorePointer#3900b
I/flutter (16625):       child: RenderBlockSemantics#d11a8
I/flutter (16625):         child: RenderExcludeSemantics#49351
I/flutter (16625):           child: RenderSemanticsGestureHandler#a8a81
I/flutter (16625):             child: RenderPointerListener#79a8c
I/flutter (16625):     onstage 4: RenderSemanticsAnnotations#697e1
I/flutter (16625):       child: RenderOffstage#9c3fe
I/flutter (16625):         child: RenderSemanticsAnnotations#86a76
I/flutter (16625):           child: RenderRepaintBoundary#b3483
I/flutter (16625):             child: RenderFractionalTranslation#4aced
I/flutter (16625):     onstage 5: RenderConstrainedBox#564ac
I/flutter (16625):     onstage 6: RenderLimitedBox#199fe
I/flutter (16625):       child: RenderConstrainedBox#b2002
I/flutter (16625):     onstage 7: RenderIgnorePointer#0c7ec NEEDS-LAYOUT NEEDS-PAINT
I/flutter (16625):       child: RenderRepaintBoundary#0e41e NEEDS-LAYOUT NEEDS-PAINT
I/flutter (16625):         child: RenderOpacity#15dd1 NEEDS-LAYOUT NEEDS-PAINT
I/flutter (16625):           child: RenderPhysicalModel#6e59a NEEDS-LAYOUT NEEDS-PAINT
I/flutter (16625):             child: _RenderInkFeatures#35c1b NEEDS-LAYOUT NEEDS-PAINT
I/flutter (16625): ════════════════════════════════════════════════════════════════════════════════════════════════════
I/flutter (16625): ══╡ EXCEPTION CAUGHT BY SCHEDULER LIBRARY ╞═════════════════════════════════════════════════════════
I/flutter (16625): The following assertion was thrown during a scheduler callback:
I/flutter (16625): Updated layout information required for RenderIgnorePointer#0c7ec NEEDS-LAYOUT NEEDS-PAINT to
I/flutter (16625): calculate semantics.
I/flutter (16625): 'package:flutter/src/rendering/object.dart':
I/flutter (16625): Failed assertion: line 2636 pos 12: '!_needsLayout'
I/flutter (16625): 
I/flutter (16625): Either the assertion indicates an error in the framework itself, or we should provide substantially
I/flutter (16625): more information in this error message to help you determine and fix the underlying cause.
I/flutter (16625): In either case, please report this assertion by filing a bug on GitHub:
I/flutter (16625):   https://github.com/flutter/flutter/issues/new?template=BUG.md
I/flutter (16625): 
I/flutter (16625): When the exception was thrown, this was the stack:
I/flutter (16625): #2      RenderObject._getSemanticsForParent (package:flutter/src/rendering/object.dart:2636:12)
I/flutter (16625): #3      RenderObject._getSemanticsForParent.<anonymous closure> (package:flutter/src/rendering/object.dart:2658:61)
I/flutter (16625): #4      _RenderTheatre.visitChildrenForSemantics (package:flutter/src/widgets/overlay.dart:757:14)
I/flutter (16625): #5      RenderObject._getSemanticsForParent (package:flutter/src/rendering/object.dart:2653:5)
I/flutter (16625): #6      RenderObject._getSemanticsForParent.<anonymous closure> (package:flutter/src/rendering/object.dart:2658:61)
I/flutter (16625): #7      RenderObjectWithChildMixin.visitChildren (package:flutter/src/rendering/object.dart:3023:14)
I/flutter (16625): #8      RenderObject.visitChildrenForSemantics (package:flutter/src/rendering/object.dart:2741:5)
I/flutter (16625): #9      RenderSemanticsAnnotations.visitChildrenForSemantics (package:flutter/src/rendering/proxy_box.dart:4489:11)
I/flutter (16625): #10     RenderObject._getSemanticsForParent (package:flutter/src/rendering/object.dart:2653:5)
I/flutter (16625): #11     RenderObject._getSemanticsForParent.<anonymous closure> (package:flutter/src/rendering/object.dart:2658:61)
I/flutter (16625): #12     RenderAbsorbPointer.visitChildrenForSemantics (package:flutter/src/rendering/proxy_box.dart:3373:14)
I/flutter (16625): #13     RenderObject._getSemanticsForParent (package:flutter/src/rendering/object.dart:2653:5)
I/flutter (16625): #14     RenderObject._getSemanticsForParent.<anonymous closure> (package:flutter/src/rendering/object.dart:2658:61)
I/flutter (16625): #15     RenderObjectWithChildMixin.visitChildren (package:flutter/src/rendering/object.dart:3023:14)
I/flutter (16625): #16     RenderObject.visitChildrenForSemantics (package:flutter/src/rendering/object.dart:2741:5)
I/flutter (16625): #17     RenderObject._getSemanticsForParent (package:flutter/src/rendering/object.dart:2653:5)
I/flutter (16625): #18     RenderObject._getSemanticsForParent.<anonymous closure> (package:flutter/src/rendering/object.dart:2658:61)
I/flutter (16625): #19     RenderObjectWithChildMixin.visitChildren (package:flutter/src/rendering/object.dart:3023:14)
I/flutter (16625): #20     RenderObject.visitChildrenForSemantics (package:flutter/src/rendering/object.dart:2741:5)
I/flutter (16625): #21     RenderObject._getSemanticsForParent (package:flutter/src/rendering/object.dart:2653:5)
I/flutter (16625): #22     RenderObject._getSemanticsForParent.<anonymous closure> (package:flutter/src/rendering/object.dart:2658:61)
I/flutter (16625): #23     RenderObjectWithChildMixin.visitChildren (package:flutter/src/rendering/object.dart:3023:14)
I/flutter (16625): #24     RenderObject.visitChildrenForSemantics (package:flutter/src/rendering/object.dart:2741:5)
I/flutter (16625): 00:46 +0: kept alive Hero does not throw when the transition begins [E]
I/flutter (16625):   Test failed. See exception logs above.
I/flutter (16625):   The test description was: kept alive Hero does not throw when the transition begins
I/flutter (16625):   
I/flutter (16625): 00:46 +0 -1: Some tests failed.
I/flutter (16625): #25     RenderSemanticsAnnotations.visitChildrenForSemantics (package:flutter/src/rendering/proxy_box.dart:4489:11)
I/flutter (16625): #26     RenderObject._getSemanticsForParent (package:flutter/src/rendering/object.dart:2653:5)
I/flutter (16625): #27     RenderObject._getSemanticsForParent.<anonymous closure> (package:flutter/src/rendering/object.dart:2658:61)
I/flutter (16625): #28     RenderObjectWithChildMixin.visitChildren (package:flutter/src/rendering/object.dart:3023:14)
I/flutter (16625): #29     RenderObject.visitChildrenForSemantics (package:flutter/src/rendering/object.dart:2741:5)
I/flutter (16625): #30     RenderSemanticsAnnotations.visitChildrenForSemantics (package:flutter/src/rendering/proxy_box.dart:4489:11)
I/flutter (16625): #31     RenderObject._getSemanticsForParent (package:flutter/src/rendering/object.dart:2653:5)
I/flutter (16625): #32     RenderObject._getSemanticsForParent.<anonymous closure> (package:flutter/src/rendering/object.dart:2658:61)
I/flutter (16625): #33     RenderObjectWithChildMixin.visitChildren (package:flutter/src/rendering/object.dart:3023:14)
I/flutter (16625): #34     RenderObject.visitChildrenForSemantics (package:flutter/src/rendering/object.dart:2741:5)
I/flutter (16625): #35     RenderObject._getSemanticsForParent (package:flutter/src/rendering/object.dart:2653:5)
I/flutter (16625): #36     RenderObject._updateSemantics (package:flutter/src/rendering/object.dart:2617:41)
I/flutter (16625): #37     PipelineOwner.flushSemantics (package:flutter/src/rendering/object.dart:1092:16)
I/flutter (16625): #38     RendererBinding.drawFrame (package:flutter/src/rendering/binding.dart:409:21)
I/flutter (16625): #39     WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:881:13)
I/flutter (16625): #40     RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:286:5)
I/flutter (16625): #41     SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1114:15)
I/flutter (16625): #42     SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:1053:9)
I/flutter (16625): #43     LiveTestWidgetsFlutterBinding.handleDrawFrame (package:flutter_test/src/binding.dart:1430:13)
I/flutter (16625): #44     SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:969:5)
I/flutter (16625): #48     _invoke (dart:ui/hooks.dart:253:10)
I/flutter (16625): #49     _drawFrame (dart:ui/hooks.dart:211:3)
I/flutter (16625): (elided 5 frames from class _AssertionError and dart:async)
I/flutter (16625): ════════════════════════════════════════════════════════════════════════════════════════════════════
I/flutter (16625): ══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞════════════════════════════════════════════════════
I/flutter (16625): The following message was thrown:
I/flutter (16625): Multiple exceptions (2) were detected during the running of the current test, and at least one was
I/flutter (16625): unexpected.
I/flutter (16625): ════════════════════════════════════════════════════════════════════════════════════════════════════
D/ViewRootImpl@1ede5e8[MainActivity](16625): ViewPostIme pointer 0
D/ViewRootImpl@1ede5e8[MainActivity](16625): ViewPostIme pointer 1
I/flutter (16625): Some possible finders for the widgets at Offset(323.8, 410.1):
I/flutter (16625):   find.byType(PrimaryScrollController)
I/flutter (16625):   find.byType(Scaffold)
I/flutter (16625):   find.byType(FadeTransition)
I/flutter (16625):   find.byType(FractionalTranslation)
I/flutter (16625):   find.byType(SlideTransition)
D/InputMethodManager(16625): prepareNavigationBarInfo() DecorView@e265fc[MainActivity]
D/InputMethodManager(16625): getNavigationBarColor() -855310
V/InputMethodManager(16625): Starting input: tba=com.example.master_flutter ic=null mNaviBarColor -855310 mIsGetNaviBarColorSuccess true , NavVisible : true , NavTrans : false
D/InputMethodManager(16625): startInputInner - Id : 0
I/InputMethodManager(16625): startInputInner - mService.startInputOrWindowGainedFocus
D/InputTransport(16625): Input channel destroyed: 'ClientS', fd=87
D/ViewRootImpl@1ede5e8[MainActivity](16625): stopped(true) old=false
D/SurfaceView(16625): windowStopped(true) false io.flutter.embedding.android.FlutterSurfaceView{7059332 V.E...... ........ 0,0-1080,2214} of ViewRootImpl@1ede5e8[MainActivity]
D/SurfaceView(16625): surfaceDestroyed callback.size 1 #1 io.flutter.embedding.android.FlutterSurfaceView{7059332 V.E...... ........ 0,0-1080,2214}
D/SurfaceView(16625): remove() io.flutter.embedding.android.FlutterSurfaceView{7059332 V.E...... ........ 0,0-1080,2214} Surface(name=SurfaceView - com.example.master_flutter/com.example.master_flutter.MainActivity@7059332@0)/@0x38568a
D/ViewRootImpl@1ede5e8[MainActivity](16625): MSG_WINDOW_FOCUS_CHANGED 0 1
D/InputMethodManager(16625): prepareNavigationBarInfo() DecorView@e265fc[MainActivity]
D/InputMethodManager(16625): getNavigationBarColor() -855310
E/ViewRootImpl@1ede5e8[MainActivity](16625): Surface is not valid.

``` 

</details> 

<details> 
<summary>flutter doctor -v</summary> 

```
[✓] Flutter (Channel master, 1.21.0-2.0.pre.95, on Mac OS X 10.15.6 19G73, locale en-GB)
    • Flutter version 1.21.0-2.0.pre.95 at /Users/tahatesser/Code/flutter_master
    • Framework revision 50b87dc6c3 (19 hours ago), 2020-07-22 11:09:59 -0700
    • Engine revision dcc9a4048d
    • Dart version 2.9.0 (build 2.9.0-21.0.dev 9dca49e71e)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
    • Android SDK at /Users/tahatesser/Code/sdk
    • Platform android-30, build-tools 30.0.1
    • ANDROID_HOME = /Users/tahatesser/Code/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.6)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.6, Build version 11E708
    • CocoaPods version 1.9.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 47.1.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.47.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.12.2

[✓] Connected device (5 available)
    • SM M305F (mobile)      • 32003c30dc19668f          • android-arm64  • Android 10 (API 29)
    • Taha’s iPhone (mobile) • 00008020-001059882212002E • ios            • iOS 13.6
    • macOS (desktop)        • macos                     • darwin-x64     • Mac OS X 10.15.6 19G73
    • Web Server (web)       • web-server                • web-javascript • Flutter Tools
    • Chrome (web)           • chrome                    • web-javascript • Google Chrome 84.0.4147.89

• No issues found!
```

</details>

**Comment by goderbauer** [comment]
@LongCatIsLooong Are you working on this?
